manifests/init.pp

```diff
@@ -243,7 +246,10 @@
   $ssl_port                                      = $rabbitmq::params::ssl_port,
   Optional[String] $ssl_interface                = undef,
   Integer $ssl_management_port                   = $rabbitmq::params::ssl_management_port,
-  Integer $ssl_stomp_port                                = $rabbitmq::params::ssl_stomp_port,
+  Optional[String]  $ssl_management_cacert       = $ssl_cacert,
```

Comment:
Not sure if this could introduce any weird behavior. This is undef by default, since the params you're referencing are undef, but if someone set `ssl_cacert` and tried to explicitly set `ssl_management_cacert` to `undef`, it wouldn't work. I think this is still probably the best and most logical behavior (to take those as defaults).